doc/api/fs.markdown

```diff
@@ -1342,7 +1342,7 @@ The synchronous version of [`fs.writeFile()`][]. Returns `undefined`.
 ## fs.writeSync(fd, data[, position[, encoding]])
 
 * `fd` {Integer}
-* `buffer` {String | Buffer}
+* `data` {String | Buffer}
```

Comment:
I will make another PR for it. It was found while testing this patch :D  Thanks!